specs/Standards/Contracts/SelfIdentification.md

```diff
@@ -0,0 +1,207 @@
+# Contract Self-Identification ([NEP-129](https://github.com/nearprotocol/NEPs/pull/129))
```

Comment:
Add proper header per [NEP-1](https://github.com/near/NEPs/blob/master/neps/nep-0001.md). See [template](https://github.com/near/NEPs/blob/master/nep-0000-template.md).

Reply:
Add Rationale and Alternative version

Reply:
Add Copyright?